sql/core/src/main/java/org/apache/spark/sql/sources/v2/ContinuousReadSupportProvider.java

```diff
@@ -39,7 +52,7 @@
    * @param options the options for the returned data source reader, which is an immutable
    *                case-insensitive string-to-string map.
    */
-  ContinuousReader createContinuousReader(
+  ContinuousReadSupport createContinuousReadSupport(
     Optional<StructType> schema,
```

Comment:
I though this was going to have two versions instead of passing the schema? FYI @jose-torres